infra/aws/terraform/cncf-k8s-infra-aws-capa-ami/hack/disable-block.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## This script is used to disable the block on public access to AMIs
+
+AMI_REGIONS="ap-south-1,eu-west-3,eu-west-2,eu-west-1,ap-northeast-2,ap-northeast-1,sa-east-1,ca-central-1,ap-southeast-1,ap-southeast-2,eu-central-1,us-east-1,us-east-2,us-west-1,us-west-2"
+
+IFS=','
+read -ra arr <<<"$AMI_REGIONS"
+
+for val in "${arr[@]}"; do
+    aws ec2 disable-image-block-public-access --region "$val"
+done
```

Comment:
we could do this with a HCL code ? cc @bryantbiggs 

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ec2_image_block_public_access

_Don't have to be in this PR. Just want to be sure we can skip bash configuration_

Reply:
I'm asking the other CAPA maintainer whether we need to continue publishing the AMIs to 15 regions. And whether we can just publish to 1 instead. If we change to just one then we can remove this completly.

But if there was a way that this could be done in HCL then that would give us options.